File: DSA/akashyadav_590014126/Day_8_Question_3.py
def reverse_string(text):
    return text[::-1]  # Slicing the text entered by the user with -1 step which reverses the string

input_string = input("Enter a string: ")#taking the string as the input by the user
reversed_string = reverse_string(input_string)#performing the reverse function on the input string
print("Reversed string:", reversed_string)#printing the reversed string

File: DSA/Gautam_Shyamnani_590013078/Day8Question3.cpp
#include <iostream>
#include <string>
#include <algorithm>
using namespace std;

int main() {
    string text;
    cout<<"Enter a string: "; 
    getline(cin, text);

    reverse(text.begin(), text.end());

    cout << "Reversed string: " << text << endl;

    return 0;
}
